src/routes/index.tsx

```diff
@@ -1,3 +1,10 @@
 export const Routes = {
+  HomeUrl: () => `/`,
+  AppUrl: () => `/app`,
+  StatisticsUrl: () => `/app/statistics`,
   SandboxUrl: (user: string) => `/sandbox/${user}`,
+  ResetPasswordUrl: (uid?: string, token?: string) =>
+    `/resetPassword/${uid && token ? ':uid/:token' : ''}`,
```

Comment:
Trzymajmy się wskazówek od Googla: https://developers.google.com/search/docs/crawling-indexing/url-structure?hl=pl

<img width="941" alt="CleanShot 2023-03-11 at 10 13 13@2x" src="https://user-images.githubusercontent.com/22201888/224475861-81908b5c-61ab-4ff7-905a-952809da2370.png">


```suggestion
  ResetPasswordUrl: (uid?: string, token?: string) =>
    `/reset-password/${uid && token ? ':uid/:token' : ''}`,
```